lib/web_ui/lib/src/ui/window.dart

```diff
@@ -252,11 +252,11 @@ class IsolateNameServer {
 SingletonFlutterWindow get window => engine.window;
 
 class FrameData {
```

Comment:
I think this is trying to implement the same API as something that is defined in the "regular" dart:ui. Since the engine doesn't use patch files, this would leave unused parameters

https://github.com/flutter/engine/blob/93f3991842fb18574aeb8e567376b02ad48fc331/lib/ui/window.dart#L874

So this might need an ignore

Reply:
oh wait, its a private constructor

Reply:
carry on